client/common/cmdline.c

```diff
@@ -2547,10 +2547,14 @@ int freerdp_client_settings_parse_command_line_arguments(rdpSettings* settings,
 
 						if (_strnicmp("AVC444", val, 7) == 0)
 						{
+							settings->GfxH264 = TRUE;
```

Comment:
Not sure if we should enable GfxH264 here as well. IIRC the ms client only sends AVC444 supported flags but sets AVC420 to unsupported during capability exchange.

Reply:
@mfleisz since we support it and it is only used with servers that actually fall back to these old caps it does not hurt.

Reply:
> ... it is only used with servers that actually fall back to these old caps ...
That's the part I am actually not sure about ... ^^
Anyways everything else lgtm